openapi/workflow_execution_service.swagger.yaml

```diff
@@ -456,6 +456,7 @@ definitions:
       - EXECUTOR_ERROR
       - SYSTEM_ERROR
       - CANCELED
+      - CANCELLING
```

Comment:
We should use a single L in "CANCELING" to match the existing "CANCELED". (Per a quick web search, both words can be spelled either way, with a single L preferred in American English and a double L in British English. So let's be consistent.)